hedera-node/src/main/java/com/hedera/services/store/TypedTokenStore.java

```diff
@@ -307,6 +312,7 @@ public Token loadPossiblyDeletedOrAutoRemovedToken(Id id) {
 	public void persistToken(Token token) {
 		final var key = token.getId().asMerkle();
 		final var mutableToken = tokens.get().getForModify(key);
+
 		mapModelChangesToMutable(token, mutableToken);
```

Comment:
You should update this method in order to reflect the changes on the `Merkle` token.